chain/cosmos/codec.go

```diff
@@ -26,6 +28,9 @@ func DefaultEncoding() simappparams.EncodingConfig {
 	ibctypes.RegisterInterfaces(cfg.InterfaceRegistry)
 	transfertypes.RegisterInterfaces(cfg.InterfaceRegistry)
 
+	demoTypes.RegisterInterfaces(cfg.InterfaceRegistry)
```

Comment:
Do you think we need these and the ica types in the default encoding? Or can we put them in an override encoding for the ICA test? For example with osmosis: 

https://github.com/strangelove-ventures/ibctest/blob/v3/examples/osmosis/osmosis.go#L10-L17

https://github.com/strangelove-ventures/ibctest/blob/f2a589c02156f2bf47382e670fde3264f024749e/examples/osmosis/cosmos_chain_expedited_proposal_test.go#L39